src/js/controller/model.js

```diff
@@ -152,6 +152,10 @@ define([
                     var visualQuality = _.extend({}, data);
                     this.mediaModel.set('visualQuality', visualQuality);
                     break;
+
+                case 'autoplayFailed':
+                    this.setMute(false);
```

Comment:
The player doesn't have the original config values once an update has been made to the mute state in the model. We always set the model's mute value to `true` on mobile when autoplaying, so this just undoes that change.